packages/hasura/metadata/databases/futarchy/tables/tables.yaml

```diff
@@ -11,20 +11,14 @@
 - "!include public_orders.yaml"
 - "!include public_organizations.yaml"
 - "!include public_prices.yaml"
-- "!include public_prices_15m.yaml"
-- "!include public_prices_1h.yaml"
-- "!include public_prices_4h.yaml"
-- "!include public_prices_5m.yaml"
 - "!include public_prices_chart_data.yaml"
 - "!include public_program_system.yaml"
 - "!include public_programs.yaml"
 - "!include public_proposal_bars.yaml"
-- "!include public_proposal_bars_4h.yaml"
 - "!include public_proposal_details.yaml"
 - "!include public_proposal_total_trade_volume.yaml"
 - "!include public_proposals.yaml"
 - "!include public_reactions.yaml"
-- "!include public_reid_data.yaml"
```

Comment:
Would this also warrant the removal of the file in question? I assume that Hasura might not do that automagically.